projects/cxxopts/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+RUN apt install -y python3
```

Comment:
Please do an `apt-get update && ...` this will break if base-builder isn't pulled recently enough.

Reply:
Is python even needed? We include it

Reply:
> Please do an `apt-get update && ...` this will break if base-builder isn't pulled recently enough.

Oh yeah good catch!

> Is python even needed? We include it

I'll double check but I'm pretty sure it's necessary. I got an error during the bazel build without it. TBH I'm not entirely sure why Bazel even needed it, it's a C++ project and doesn't use any of the rules_python stuff.

Reply:
> Is python even needed? We include it

For reference, this is the error I get when building without python3 installed from apt. 

```
ERROR: /src/cxxopts/BUILD.bazel:12:13: Action cxxopts_fuzz_test_corpus failed: (Exit 127): make_corpus_dir failed: error executing command (from target //:cxxopts_fuzz_test_corpus)
  (cd /root/.cache/bazel/_bazel_root/35922453f9ff32ffdf58e7d54735da23/execroot/__main__ && \
  exec env - \
  bazel-out/k8-opt-exec-2B5CBBC6/bin/external/rules_fuzzing/fuzzing/tools/make_corpus_dir '--output_dir=bazel-out/k8-opt/bin/cxxopts_fuzz_test_corpus' '--corpus_list_file=bazel-out/k8-opt/bin/cxxopts_fuzz_test_corpus-0.params')
# Configuration: 6b28a955ca9d10ba9520a0c2e6c08971904abfa7923f0f46ed980b9a0896fb1d
# Execution platform: @local_config_platform//:host
/usr/bin/env: 'python3': No such file or directory
Target //:cxxopts_fuzz_test_oss_fuzz failed to build
INFO: Elapsed time: 7.691s, Critical Path: 0.17s
INFO: 14 processes: 14 internal.
FAILED: Build did NOT complete successfully
ERROR:root:Building fuzzers failed.
```

Reply:
Not sure why Bazel isn't finding the version of python that's already installed though. My guess is that it can't find python at `/usr/bin/python3`.

```
$ docker run -it --rm gcr.io/oss-fuzz-base/base-builder /bin/whereis python3
python3: /usr/local/bin/python3.8 /usr/local/bin/python3 /usr/local/bin/python3.8-config /usr/local/lib/python3.8
```

Reply:
Nevermind don't remove this please. THere's a known issue here: https://github.com/google/oss-fuzz/issues/8915